interface/reports/reports_controllers/AppointmentsController.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/*
+ * These functions are common functions used in the Appointments reports. They have pulled out 
+ * and placed in this file. This is done to prepare the for building a
+ * report generator.
+ *
+ * Copyright (C) 2018 Tigpezeghe Rodrige <tigrodrige@gmail.com>
+ * Copyright (C) 2015-2017 Terry Hill <teryhill@librehealth.io>
+ * Copyright (C) 2005-2010 Rod Roark <rod@sunsetsystems.com>
```

Comment:
Attribute the original functions (which are not individually copyright eligible) and make the copyright yours.  Rod and Terry stole things from all over the place too.  This is a white page file.  It is yours.

Reply:
The next PR will carry these copyright changes.